### DISCUSSION TITLE
Contributing the project

### DISCUSSION THREAD

**Discussion by TunahanCATAK** [munich-quantum-toolkit/core#427] [General]
Hey all, 

I'd like to contribute the project but do you have any documents to show how to prepare the local and build the project and helpful other documentations for contributions. 

Thanks in advance. 

Tuna

**Comment by burgholzer** [comment]
Hey 👋🏼 

Thanks for your interest in contributing to the project!
I'd say your best bet for starting out is reading the `Contributing` document here: https://github.com/cda-tum/mqt-core/blob/main/.github/contributing.md

We are still in the process of properly setting up the documentation for the project here.
I'll update this thread once we are done with that. I might find some time next week.
But if you wanna know more about how to set up a development environment, you can have a look at https://mqt.readthedocs.io/projects/qcec/en/latest/DevelopmentGuide.html to get some inspiration.

Other than that, I think the easiest thing is just to browse the list of open issues, especially looking out for stuff that's listed as a "good first issue". And don't hesitate to ask any questions around here!

**Reply by TunahanCATAK** [reply]
Thanks for the reply! I will try to follow the development guide this week. 

I can also note down which steps are missing in the guide so it can maybe help you to set up a proper documentation. I will let you know my findings.
